src/core/blockIntervalPointer.ts

```diff
@@ -0,0 +1,17 @@
+import { BigIntish } from '@brinkninja/types';
+import { soliditySha3 } from 'web3-utils';
+
+export type blockIntervalPointerArgs = {
+  id: BigIntish
+}
+
+const blockIntervalPointer = ({
+  id
+}: blockIntervalPointerArgs): string | null => {
+  return soliditySha3(
+    { type: 'uint64', value: id.toString() },
+    { type: 'string', value: "blockInterval" }
```

Comment:
to stay consistent with the rest of this repo, change all `"` to `'`. I should really have a linter for this...